pkgs/tools/misc/ncdu/default.nix

```
@@ -2,23 +2,23 @@
 , stdenv
 , fetchurl
 , ncurses
-, zig_0_11
+, zig_0_12
```

```suggestion
, zig
```

Use the top-level to change the default.

---

OK, time for another treewide fixup...

---

Oh hell!

Time for another treewide fixup...

P.S.:

Explaining a bit:

In the foreseeable future all the packages will be migrated to by-name hierarchy.

Pinning inputs inside expressions this way complicates the override interface, as explained in the by-name manual:

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults